.github/workflows/ruby.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
```

Comment:
Would you mind to add tests for `overcommit --run` and bakerman?

Reply:
done

Reply:
did you mean breakman? this is run by overcommit already

Reply:
not with run. breakman is executed just as a pre-push-action in overcommit which is not executed with `--run`

Reply:
ok, thanks for clarification. will add ...